website/docs/md_components/popup_components/usage_markdown.md

```diff
@@ -1,15 +1,15 @@
 A popup (also known as an overlay) is a window or dialog that appears on top of the page content.
 A popup can be classified according to two dimensions:
-- Whether the user can interact with the rest of the page, and it is called Modal when the content on the page is disabled until the user explicitly interacts with the overlay or Nonmodal when users can still interact with the background content (for example, by selecting links or tapping buttons) while the overlay remains visible.
-- Whether the background is dimmed the popup is called a lightbox. 
+- whether the user can interact with the rest of the page, and it is called Modal when the content on the page is disabled until the user explicitly interacts with the overlay or Nonmodal when users can still interact with the background content (for example, by selecting links or tapping buttons) while the overlay remains visible
```

Comment:
Modal should be wrapped with backtics ` so that it is clear that it's an identifier